frontend/src/pages/CardSearchPage.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components/macro";
+import useCardSearch from "../hooks/useCardSearch";
+import CardSearchResult from "../components/CardSearchResult";
+import CardSearch from "../components/CardSearch";
+
+
```

Comment:
I would use prettier to format all files

Reply:
thanks, I will consider this for the next pr